deploy/module_manifest.py

```
@@ -124,6 +122,6 @@
     from soundcard.mediafoundation import SoundcardRuntimeWarning
     from threading import Event, Lock
     from types import ModuleType
-    from typing import Any, Callable, ClassVar, Dict, IO, Iterable, List, Optional, Set, TYPE_CHECKING, Tuple, Type, TypeVar, Union, cast
+    from typing import Any, Callable, ClassVar, Dict, IO, Iterable, List, NamedTuple, Optional, Set, TYPE_CHECKING, Tuple, Type, TypeVar, Union, cast
```

Ruff (0.15.2)

[error] Redefinition of unused `Callable` from line 77: `Callable` redefined here

(F811)